src/interfaces/chain.cpp

```diff
@@ -248,6 +249,20 @@ class ChainImpl : public Chain
         // LockImpl to Lock pointer
         return std::move(result);
     }
+    Optional<bool> filterMatchesAny(const uint256& hash, const GCSFilter::ElementSet& filter_set) override
+    {
+        const BlockFilterIndex* block_filter_index = GetBlockFilterIndex(BlockFilterType::BASIC);
```

Comment:
Looks like the pointer itself could be even const too: 
const BlockFilterIndex* **const** block_filter_index = ...